webpack/assets/javascripts/react_app/components/notifications/Notification.js

```diff
@@ -18,6 +18,33 @@ const Notification = ({created_at, seen, text, level, id}) => {
     (<strong {...tooltip} className="drawer-pf-notification-message pointer"
              onClick={markAsRead}>{text}</strong>);
 
+  const links = [];
+
+  let actions_dropdown = '';
+
+  if (actions.links != null) {
+    for (let i = 0; i < actions.links.length; i++) {
+      let li_key = 'notification-link-' + i + '-' + id;
+
+      links.push((<li key={li_key} id={li_key}>
+                    <a target="_blank" href={actions.links[i].href}>
+                    {actions.links[i].title}</a>
+                  </li>
+                 ));
+    }
+
+    actions_dropdown = (
+      <div className="dropdown pull-right dropdown-kebab-pf">
+        <button className="btn btn-link dropdown-toggle" type="button" id="dropdownKebabRight{id}"
```

Comment:
In patternfly, the kebab is a restyled version of a dropdown supporting the same features as the bootstrap dropdown component. I'll check with Roxanne as to whether there are use cases for the dropdown menu or whether the kebab should always be used.